### PR TITLE
Works Detailed pop-up

### PR DESCRIPTION
**Pull request by mohashyne**
# New Features Added

- [ ]  Add the HTML structure for a detailed pop-up
- [ ]  Add Project details in an array object
- [ ] Update styling, HTML & JavaScript function
- [ ] Add function to append cards dynamically
- [ ] Add function to pop-up cards details dynamically

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the javascript linters also setup, for this project both the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters should be configured
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by mohashyne** [comment]
> You have submitted a project, that does not have the javascript linters also setup, for this project both the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters should be configured Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

Thank you for bringing up the issue regarding the JavaScript linters in the submitted project. I apologize for the oversight. I want to assure you that all the linters, including JavaScript and HTML&CSS, have been properly configured for this project. However, it seems there may have been a misunderstanding or a missed observation.

To ensure everything is in order, I kindly request you to thoroughly review the project again. I believe you'll find that the linters are indeed set up correctly. If you still encounter any issues or have further questions, please don't hesitate to ask. Feel free to tag me in your questions or comments to ensure I receive the notification promptly.

Please accept my apologies for any confusion caused, and I appreciate your patience and understanding. 

Best regards,
Muhammad A Salihu

File: dist/css/style.css [hunk 1]
```diff
@@ -867,6 +868,13 @@ h1 {
...
+  .btn--green {
+    background-color: #fff;
+    border: 1px solid #396df2;
+    border-radius: 8px;
+    margin-left: 20px;
+  }
+
 /* Skills */
 .about-self {
   width:  5rem;
```
Comment (usorfaitheloho) [inline dist/css/style.css:880]: Please kindly implement the mobile view of the popup as it is shown in the Figma template, you are to ensure your design is responsive and looks as close as possible to the Figma template given for the mobile screen 😄 

Figma template :
image

Your design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @mohashyne  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @mohashyne  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/css/style.css (hunk 1 above)
Comment (mohashyne) [reply dist/css/style.css:880]: # changes implemented

The mobile view pop-up is now showing correctly as you requested. 👍

**Comment by mohashyne** [comment]
# And Finally

The underline in the "see live" and "see source" buttons of the desktop view has been removed as requested 
😄

File: dist/css/style.css (hunk 1 above)
Comment (1GORDON) [inline dist/css/style.css:880]: Great design team🤝

- [ ] As requested by the previous reviewer, kindly ensure your mobile popup is a replica of the design given. Your mobile popup should have a close button to close the popup window.
- [ ] Kindly ensure your mobile popup has all features just like in the Figma design given. Your design seems to be hiding the buttons and the `Tonic` feature at the top of your design.
- Your design
Your Mobile popup
- Given design
Given mobile popup

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @mohashyne & @pulgosodog ✋

Good job so far!

## Highlights!
- Followed GitHub Flow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/css/style.css (hunk 1 above)
Comment (mohashyne) [reply dist/css/style.css:880]: # Required Changes Implemented

Close button was fixed and the required changes has been implemented

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @mohashyne 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**